### ISSUE TITLE
Importer: Validate that the OpenAI API key works before trying

### ISSUE THREAD

**Issue by eric-anderson** [aryn-ai/sycamore#162]
If a non-functional OpenAI API key is used, the importer will report that we are getting throttled errors. That makes it hard to debug why importing is not working and worse the error message is merged into the middle of tons of ray output.

Fix it so that we check the that the key works before starting and abort like we do if the key is missing.

**Comment by HenryL27** [comment]
I'm pretty sure this is done
